ssh-key/src/public.rs

```diff
@@ -78,7 +78,7 @@ use std::{fs, path::Path};
 /// The serialization uses a binary encoding with binary formats like bincode
 /// and CBOR, and the OpenSSH string serialization when used with
 /// human-readable formats like JSON and TOML.
-#[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Ord)]
+#[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Ord, Hash)]
```

Comment:
These trait impls are alphabetized by trait name. Can you follow that convention? (and in the other places it was added)

```suggestion
#[derive(Clone, Debug, Eq, Hash, PartialEq, PartialOrd, Ord)]
```

Reply:
Sure, rebased updated the diff now